db_facts/lpass.py

```diff
@@ -19,20 +19,16 @@ def pull_lastpass_aws_iam(lastpass_entry_name: str) -> LastPassAWSIAM:
 
 
 def lpass_field(name: str, field: str) -> str:
-    if field == 'notes':
-        field_arg = '--notes'
-    elif field == 'username':
-        field_arg = '--username'
-    elif field == 'password':
-        field_arg = '--password'
-    elif field == 'url':
-        field_arg = '--url'
-    else:
-        field_arg = '--field=' + field
-    raw_output = check_output(['lpass',
-                               'show',
-                               field_arg,
-                               name])
+    # This used to use the lastpass-cli to pull credentials. But we've moved
+    # from lastpass to 1password. This command retrieves the fields in the
+    # same format from 1password instead.
+
```

Comment:
Can you make this comment even more explicit? I'd like it to make clear that ALL functions referencing LastPass/lpass are being redirected to pull data from 1Password instead. The names have not been changed in the interest of time. This needs to be an **unmissable comment** for anyone working on this in the future, so you can go wild with all caps.